.vscode/launch.json

```diff
@@ -99,6 +99,27 @@
         },
         {
             "name": "Web Tests",
+            "type": "extensionHost",
+            "debugWebWorkerHost": true,
+            "request": "launch",
+            "args": [
+                "${workspaceFolder}/src/test/datascience",
+                "--extensionDevelopmentPath=${workspaceFolder}",
+                "--enable-proposed-api",
+                "--extensionDevelopmentKind=web",
+                "--extensionTestsPath=${workspaceFolder}/out/extension.web.bundle"
+            ],
+            "outFiles": ["${workspaceFolder}/out/**/*.*"],
+            "sourceMaps": true,
+            "preLaunchTask": "Start Jupyter Server",

```

Comment:
The way the launch task works is:
* Start jupyter server, update setting to store url and update a text file to store the pid of jupyter server.
* Start debugging
* Stop jupyter server (pid of server is stored in text file)